### PR TITLE
GitHub Actions workflow for PhpMetrics static analysis

### LINKED ISSUE #18: phpmetrics workflow
Add a GitHub workflow for [phpmetrics](https://github.com/phpmetrics/PhpMetrics) that uploads artifacts to a GitHub page.

### PR DESCRIPTION
**Pull request by maluueu**
Closes #18

### CODE REVIEW

**Comment by calvinalkan** [comment]
Please configure it correctly:
https://github.com/phpmetrics/PhpMetrics#configure

File: .github/workflows/phpmetrics.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
+name: Deploy PhpMetrics to GitHub Pages
+
+on:
+  push:
```
Comment (calvinalkan) [inline .github/workflows/phpmetrics.yml:4]: @maluueu Shouldn't this be set to tag?

Comment (maluueu) [reply .github/workflows/phpmetrics.yml:4]: We can either leave it at push so PhpMetrics always represents the latest changes (e.g. master branch) or change it to tag, so it always is coupled to the latest released tag.

Comment (calvinalkan) [reply .github/workflows/phpmetrics.yml:4]: I prefer latest tag

File: phpmetrics.json [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    "exclude": [
+      "tests"
+    ],
+    "report": {
+        "html": "htmlout",
+        "csv": "/tmp/report.csv",
+        "json": "/tmp/report.json",
+        "violations": "/tmp/violations.xml"
+    },
+    "groups": [
```
Comment (calvinalkan) [inline phpmetrics.json:14]: Please use readable names for the "name" key.

Either the namespace suffix, ie: SessionPsr16 or the directory name, ie: session-psr16

**Review by calvinalkan — CHANGES_REQUESTED** [review]

File: phpmetrics.json [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+        {
```
Comment (calvinalkan) [inline phpmetrics.json:15]: Please add the better-wp-package aswell

File: phpmetrics.json (hunk 2 above)
Comment (maluueu) [reply phpmetrics.json:14]: All names have been changed to reflect the namespace suffix.

File: phpmetrics.json (hunk 3 above)
Comment (maluueu) [reply phpmetrics.json:15]: BetterWPDB has been added, ScopableWP has been renamed and the deprecated ParameterBag has been removed

File: phpmetrics.json [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+            "name": "HttpRouting",
+            "match": "!Snicco\\\\Component\\\\HttpRouting\\\\.*!"
+        },
+        {
+            "name": "Psr7ErrorHandler",
+            "match": "!Snicco\\\\Component\\\\Psr7ErrorHandler\\\\.*!"
+        },
+        {
+            "name": "ScopableWP",
```
Comment (calvinalkan) [inline phpmetrics.json:80]: wrong.

**Review by calvinalkan — CHANGES_REQUESTED** [review]

File: phpmetrics.json (hunk 4 above)
Comment (calvinalkan) [reply phpmetrics.json:80]: Rebase this into one commit please

Comment (maluueu) [reply phpmetrics.json]: done

**Review by calvinalkan — DISMISSED** [review]

**Pull request merged**